website/docs/d/cloudfront_distribution.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "CloudFront"
+layout: "aws"
+page_title: "AWS: aws_cloudfront_distribution"
+description: |-
+  Provides a CloudFront web distribution data source.
+---
+
+# Data source: aws_cloudfront_distribution
+
+Use this data source to retrieve information about a CloudFront distribution.
+
+## Example Usage
+
+```hcl
+data "aws_cloudfront_distribution" "test" {
+  id = "${aws_cloudfront_distribution.s3_distribution.id}"
```

Comment:
We typically would not recommend chaining a resource into a data source. This should be revised to a hardcoded ID argument. 👍 

```suggestion
  id = "EDFDVBD632BHDS5"
```